src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -574,21 +575,41 @@ def handleSpecialCERNMergeSettings(self, funcName):
         Enable lazy-download for fastCloning for all CMSSW_7_5 jobs (currently off)
         Enable lazy-download for all merge jobs
         """
-        cmsswVersion = os.environ['CMSSW_VERSION']
+        cmsswVersion = os.environ.get('CMSSW_VERSION', "")
         if cmsswVersion.startswith("CMSSW_7_5") and False:
             print "Using fastCloning/lazydownload"
-            self.process.add_(
-                cms.Service("SiteLocalConfigService",
-                            overrideSourceCloneCacheHintDir=cms.untracked.string("lazy-download")
-                            )
-                )
+            self.process.add_(cms.Service("SiteLocalConfigService",
+                                          overrideSourceCloneCacheHintDir = cms.untracked.string("lazy-download")))
         elif funcName == "merge":
             print "Using lazydownload"
-            self.process.add_(
-                cms.Service("SiteLocalConfigService",
-                            overrideSourceCacheHintDir=cms.untracked.string("lazy-download")
-                            )
-                )
+            self.process.add_(cms.Service("SiteLocalConfigService",
+                                          overrideSourceCacheHintDir = cms.untracked.string("lazy-download")))
+        return
+
+    def handleCondorStatusService(self):
+        """
+        _handleCondorStatusService_
+
+        Enable CondorStatusService for CMSSW releases that support it.
+
```

Comment:
Hi Dirk, so following are supported? (7_5_0 is not supported)?
If there is document available could you add the link on the comment, just for the reference?
```
    CMSSW_8_*_* and up 
     CMSSW_7_6_* and up 
     CMSSW_7_5_1 and up
     CMSSW_7_4 _7 and up
```

Maybe you could have condition like befow more clarity

``` python
verson = (result.group(1) *100 + result.group(2)* 10 + result.group(3))
if  version >= 747 and version != 750: 
    addCondorStatusService = True
```